ios/Handlers/AuthenticatorsHandler.swift

```diff
@@ -1,6 +1,6 @@
 protocol BridgeToAuthenticatorsHandlerProtocol: AnyObject {
     func registerAuthenticator(_ userProfile: ONGUserProfile,_ authenticatorType: ONGAuthenticatorType, _ completion: @escaping (Bool, Error?) -> Void)
-    func deregisterAuthenticator(_ userProfile: ONGUserProfile, _ authenticatorId: ONGAuthenticatorType, _ completion: @escaping (Bool, Error?) -> Void)
+    func deregisterAuthenticator(_ userProfile: ONGUserProfile, _ authenticatorId: String, _ completion: @escaping (Bool, Error?) -> Void)
```

Comment:
If you resign from strong type `ONGAuthenticatorType` I suggest to create your own Swift enum, not generic String.

Reply:
Btw why don't you use `Result<SomeType, Error>` or just `Error?`? I guess you don't need this `Bool`.